app/helpers/sessions_helper.rb

```diff
@@ -3,15 +3,36 @@ def log_in user
     session[:user_id] = user.id
   end
 
+  def remember user
+    user.remember
+    cookies.permanent.signed[:user_id] = user.id
```

Comment:
cái cookie này nó sống đến khi nào vậy?

Reply:
khi khởi động lại trình duyệt cũng như khởi động lại toàn bộ hệ thống.

Reply:
k đúng, e tìm hiểu thêm nhé

Reply:
nó tồn tại cho đến khi cửa sổ trình duyệt sử dụng cookies bị đóng. Tuy nhiên người ta có thể thiết lập tham số thời gian để cookie có thể sống lâu hơn (6 tháng chẳng hạn)